### DISCUSSION TITLE
Ethos Suite for Linux

### DISCUSSION THREAD

**Discussion by BigChinoDon** [FrSkyRC/ETHOS-Feedback-Community#3201] [Ideas]
Is a Linux version of the Ethos Suite in the works?

**Comment by Vance1122** [comment]
I sure hope so. I installed windows 10 in a Virtualbox session. Ethos Suite runs in that, but it is a pain to manage the USB connections. When you plug the radio in, it will auto mount 2 drives called flash and the other is the name of the radio. You have to unmount them, then in virtualbox at the bottom of the window there is a USB icon. You right click that and select your radio as the device you want to connect. Then Ethos Suite will detect your radio. It is buggy, but for the most part it works.

**Comment by FireflyJar** [comment]
I am switching to Linux too, so fed up with Windows.  Docker might be a temporary solution as well.  I don't know enough to do it myself.

**Comment by Crunch69** [comment]
Android version would be great Window sucks after Win 7

**Reply by andika207** [reply]
the ETHOS suite doesn't support W7 anymore.
XP will be my primary OS forever and I will ditch W7 because I never liked it as soon as W11 LTSC comes out

**Comment by Vance1122** [comment]
A Linux version could be ported to Android since Google based Android on
linux.

**Comment by dspanheimerrobo** [comment]
I also want a Linux version... Just want to make sure the developers understand that there really are several Linux users...

**Comment by jeffgrain** [comment]
Yes.. I did mention this about a year ago.. never going back to windows ... and I would like to see more support

**Reply by robthomson** [reply]
I think the real challenge here is dev time vs return for what is quite a small market. 

Never say never.. But it's probably quite low on list of priorities.

**Reply by jeffgrain** [reply]
And I agree with that.. it is dev time.. but the linux world is growing..  and more so as windows becomes more ad based rather than a gui..  the linux train is starting to move.. best get aboard before the train leaves without you..

**Comment by Vance1122** [comment]
They have Ethos Suite for MacOS. It would not be that much more effort as macos and Linux are both based on Unix.

**Comment by FireflyJar** [comment]
more manufactures are offering PCs with Linux, its cheaper for them as well, to many support requests to deal with Win11 issues.
I got Ethos suite running in "Winboat" on Linux Fedora, seems to work ok, winboat is very new, it should get better.

**Comment by Vance1122** [comment]
That is awesome. I know there is a new distro of Linux that includes
WinBoat built in. Can't remember it's name, but it could help a lot of
people make the migration to Linux. Would still be nice to have a Linux
native Ethos Suite app though.

**Comment by etymotic** [comment]
Ethos Suite will run using Wine, but you can't do anything with it. It doesn't detect the radio plugged in to USB. 

I get around with with a windows virtual machine, but that's not a reasonable solution for most people.

Some of our club members fell for a windows malware scam recently, so there has been a push to get the old folks to ditch windows and try Linux Mint. It's hard to sell if they can't run their RC transmitter software.

**Comment by Vance1122** [comment]
There is a software now that allows running windows software on Linux and allows USB pass through.  It is called winboat. It is beta right now, but works ok.

**Comment by Vance1122** [comment]
I can confirm that Ethos Suite for windows works through Winboat on linux
mint. You have to configure USB pass through on Winboat and then Ethos
Suite will connect to your radio. I was able to back up my models, update
Ehtos on the radio and update the rf module to the latest firmware. I have
the X20RS.
